gnomad_qc/v5/data_ingestion/field_md_to_html.py

```diff
@@ -20,13 +22,18 @@
 first_table_processed = False
 second_table_processed = False
 
-elements = soup.find_all(["p", "table"])
+elements = soup.find_all(["p", "table", "h1", "h2"])
 
 paragraphs_before_first_table = []
 paragraphs_before_second_table = []
 
+current_header = None
+
 for element in elements:
-    if element.name == "p":
+    if element.name in ["h1", "h2"]:
+        current_header = str(element)
```

Comment:
naive question: could you directly add the header and paragraph elements to the final output (since the markdown converter and BeautifulSoup seem to be storing the contents in the same order as the markdown)? something like:
```
for element in elements:
    if element.name in ["h1", "h2"]:
        final_sections.append(str(element))
    elif element.name == "p":
        final_sections.append(str(element))
    elif element.name == "table":
        # Build stylized table
        # same code here
```
just to remove the extra variables `current_header`, `paragraphs_before_first_table`, `first_table_processed`, `paragraphs_before_second_table`, `second_table_processed`